client/src/components/CaptureMatching/CurrentCaptureNumber.js

```diff
@@ -0,0 +1,37 @@
+import React, {useState} from 'react'
+
+import Data from './CaptureData.json'
+
+import {Paper, Typography, Box, Icon} from '@material-ui/core';
+
+
+ const treeList = Data.trees
+
+function CurrentCaptureNumber(props) {
```

Comment:
I'd recommend you change the component name and some of the props to be more generic, since this is being used for captures and for candidate trees.